kafka-connect/kafka-connect-runtime/main/LICENSE

```diff
@@ -1448,3 +1448,51 @@ Project URL (from POM): https://github.com/awslabs/aws-eventstream-java
 License (from POM): Apache License, Version 2.0 - https://aws.amazon.com/apache2.0
 
 --------------------------------------------------------------------------------
```

Comment:
We need to add the incubator jar, similar to the comments in https://github.com/apache/iceberg/pull/12365